learn/src/main/java/org/navgurukul/learn/ui/learn/viewholder/QuestionCodeCourseViewHolder.kt

```diff
@@ -23,10 +23,8 @@ BaseCourseViewHolder(itemview){
     fun bindView(item: QuestionCodeBaseCourseContent, callback: (BaseCourseContent) -> Unit) {
         super.bind(item)
 
-        codeBody.text = HtmlCompat.fromHtml(
-            item.value
-                ?: "", HtmlCompat.FROM_HTML_MODE_COMPACT
-        )
+        codeBody.text = item.value?.replace("<br>", "\n")?.replace("&emsp;", " ")
```

Comment:
since '\<br>' , \n etc. are used in many other pplaces as well for similar logic, then let's create a constant string to ensure no redundant code is there and it becomes easy to modify at single place.